CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### API Breaking
 
 * (ante) [#1214](https://github.com/evmos/ethermint/pull/1214) Set mempool priority to EVM transactions.
+* (keeper) [#1405](https://github.com/evmos/ethermint/pull/1405) Remove `WithChainID`and `ChainID` methods from evm keeper, please use `ethermint.ParseChainID` to parse from context.
```

Comment:
this is unrelated to the scope of the PR, please create a separate issue if you think this should be removed

Reply:
reverted